### ISSUE TITLE
VAR_INVALID_NAME false positive with number-suffixed names

### ISSUE THREAD

**Issue by timbeadle** [cflint/CFLint#532]
```cfml
if (urlScope.keyExists('id2') && isNumeric(urlScope.id2)) {
	route &= '/:id2';
}
```

`[INFO]: [cflint] VAR_INVALID_NAME: Variable id2 is not a valid name. Please use camelCase or underscores.`

Using CFLint v1.3.0.

**Comment by ryaneberly** [comment]
Does :id2 dereference id2 automatically?

**Comment by KamasamaK** [comment]
That is just concatenating a string literal, but that would be beside the point. The issue is simply that variable names like the example above or `address1` are triggering the *_INVALID_NAME rules.

**Comment by ryaneberly** [comment]
Thanks @KamasamaK ,  I agree, i was just curios.

**Comment by ryaneberly** [comment]
This has been a 'feature' almost since the beginning. I had to disabled or change almost a dozen tests after I fixed this one.  :)  However, I agree that ending a number should not be considered invalid naming.

**Comment by timbeadle** [comment]
Thanks @ryaneberly for the quick turnaround 😄
